auraloss/freq.py

```diff
@@ -181,7 +184,10 @@ def __init__(
                 raise ValueError(
                     f"`sample_rate` must be supplied when `perceptual_weighting = True`."
                 )
-            self.prefilter = FIRFilter(filter_type="aw", fs=sample_rate)
+            if self.prefilter is None:
+                self.prefilter = FIRFilter(filter_type="aw", fs=sample_rate)
+            else:
+                self.prefilter = FIRSequential(FIRFilter(filter_type="aw", fs=sample_rate), self.prefilter)
```

Comment:
Following the above, I think this means we can simplify this by removing the need for `FIRSequential`. To achieve this we could think of having two separate prefilter attributes. For example, for the user specified prefilter, `self.user_prefilter` and for the perceptual weighting we could set `self.prefilter`, as it is now. Then, in the `forward()` we can check for either and apply if needed. How does this sound?